### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -7,41 +7,6 @@ NOTICES:
 This document includes a list of open source components used in Mattermost Mobile, including those that have been modified.
 
 --------
-## @babel/runtime
```
Comment (enahum) [inline NOTICE.txt:10]: this is still being used

**Review by enahum — CHANGES_REQUESTED** [review]
Seems we can close this PR

File: NOTICE.txt (hunk 1 above)
Comment (amyblais) [reply NOTICE.txt:10]: We don't list devDependencies in the NOTICE.txt.

**Review by enahum — APPROVED** [review]

**Pull request merged**